### ISSUE TITLE
 [Bug] Anthropic CORS error

### ISSUE THREAD

**Issue by kylix31** [ChatGPTNextWeb/ChatGPT-Next-Web#5452]
### 📦 Deployment Method

Official installation package

### 📌 Version

2.15.2

### 💻 Operating System

Other Linux

### 📌 System Version

Arch 6.10.9-zen1-2-zen

### 🌐 Browser

Other

### 📌 Browser Version

Electron

### 🐛 Bug Description

When I try to use any Anthropic model, I catch this error:

```json
{
  "type": "error",
  "error": {
    "type": "authentication_error",
    "message": "CORS requests must set 'anthropic-dangerous-direct-browser-access' header"
  }
}
```

**Comment by QAbot-zh** [comment]
更新版本测试了吗，我记得之前就有类似的帖子似乎是解决了的

https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/issues/5429

**Comment by Blackmesa-Canteen** [comment]
Not fixed yet in Version 2.15.2 (20240913.094417). Error message:
```
{
  "type": "error",
  "error": {
    "type": "authentication_error",
    "message": "CORS requests must set 'anthropic-dangerous-direct-browser-access' header"
  }
}

```

**Comment by sunilsoni** [comment]
Anyone knows how to fix it?

**Comment by lloydzhou** [comment]
using tauri http API replace fetch in app，can fix CORS error

**Comment by sunilsoni** [comment]
> using tauri http API replace fetch in app，can fix CORS error

In Mac App where can i do this settings? can you share steps or more details ?

**Comment by andremacola** [comment]
Still not fixed on  2.15.3

**Comment by m0rg5** [comment]
> Still not fixed on 2.15.3

Yep, same here.

**Comment by lloydzhou** [comment]
will fix in #5565
